test/CadlRanchProjects/azure/core/traits/src/Generated/TraitsClient.cs

```diff
@@ -319,7 +319,7 @@ internal HttpMessage CreateRepeatableActionRequest(int id, RequestContent conten
             }
             if (repeatabilityFirstSent != null)
             {
-                request.Headers.Add("Repeatability-First-Sent", repeatabilityFirstSent.Value, "O");
+                request.Headers.Add("Repeatability-First-Sent", repeatabilityFirstSent.Value, "R");
```

Comment:
Is this expected change?

Reply:
Yes. The default encoding for DateTime in head is rfc7231 whose format is `R`